helm/sealed-secrets/templates/role.yaml

```diff
@@ -37,6 +37,10 @@ metadata:
 rules:
   - apiGroups:
       - ""
+    resourceNames:
+      - 'http:{{ include "sealed-secrets.fullname" . }}:'
+      - 'http:{{ include "sealed-secrets.fullname" . }}:http'
```

Comment:
I think these two are not needed, only `{{ include "sealed-secrets.fullname" . }}`. This would align it with the `controller.jsonnet` file we mention.

`kubeseal` needs to fetch the service to get the port name (in our case `http`) that is later used through the proxy, that's why the `services/proxy` resource includes them.

Reply:
I'm not sure why I put in the other two lines. I guess it was a copy paste mistake.
Fixed it and bumped the patch version for the chart